.github/workflows/ci.yaml

```diff
@@ -0,0 +1,19 @@
+on: [push, pull_request]
```

Comment:
I don't think we gain much having a github action for this now since we need to do the jenkins build to deploy the gem. I'd defer doing this until we resolve https://trello.com/c/BrOSVOpO/151-what-do-we-do-about-repos-that-auto-deploy-using-travis

Reply:
I've removed that commit